gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -607,6 +633,7 @@ message ListUserApprovalsRequest {
   string q = 8;
   repeated string account_types = 9;
   repeated string resource_types = 10;
+  bool stale = 11;
```

Comment:
we don't need this filter as stale approvals won't be actionable and not relevant for listing user approvals request

Reply:
this stale is the flag to fetch stale approvals if needed, don't we need it?